docs/cover.md

```diff
@@ -1,57 +1,71 @@
-# Code coverage for GoAWK
 
-GoAWK implements code coverage functionality, similar to the one built in the Golang itself (https://go.dev/blog/cover).
+# GoAWK's code coverage feature
 
-![screenshot](cover.png)
+GoAWK implements code coverage functionality, similar to the [code coverage built into Go](https://go.dev/blog/cover). This feature was implemented by Volodymyr Gubarkov ([xonixx](https://github.com/xonixx)), primarily in [PR #154](https://github.com/benhoyt/goawk/pull/154).
```

Comment:
I'd like to give you credit. :-) Let me know if you want your name or profile link tweaked.

Reply:
This is fine, thanks!